_data/engine-cli/docker_ps.yaml

```diff
@@ -348,7 +348,6 @@ examples: |-
   ```
 
   The following filter matches all containers that have exposed TCP port in the range of `8000-8080`:
-
```

Comment:
@tiborvass the source file is correct. Maybe the YAML generator is eating the newlines?